src/spi.rs

```diff
@@ -116,6 +116,15 @@ pub struct Spi<SPI, REMAP, PINS, FRAMESIZE> {
     _framesize: PhantomData<FRAMESIZE>,
 }
 
+/// The frame format to send the data in
+#[derive(Debug, Clone, Copy)]
+pub enum SpiFrameFormat {
+    /// Least significant byte first
+    Lsb,
+    /// Most significant byte first
+    Msb,
+}
+
```

Comment:
These comments should say `Most significant bit first` I believe.

Also, I almost want to call the members `[ML]sbFirst` but that becomes very long.

Reply:
yeah naming these is kind of tricky because if you want to spell it out so its obvious it becomes super long, but otherwise its kind of awkward. perhaps `MostSigBitFirst`|`LeastSigBitFirst`? "sig" feels awkward but its Close Enough:tm: I think, thoughts?

Reply:
I think LSB and MSB are well known enough that we can call them `LsbFirst` and `MsbFirst`